src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -524,23 +522,28 @@ public void primaryKeyTest() throws Exception {
 
     assertThat(
         "single-column primary keys are set correctly",
-        variants.getPrimaryKey().stream().map(Column::getName).collect(Collectors.toList()),
-        equalTo(Collections.singletonList("id")));
+        variants.getPrimaryKey(),
+        containsInAnyOrder(variants.getColumnByName("id").get()));
```

Comment:
I think it's a good sign that these tests became more clear as a result of these changes :)